sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -1184,6 +1184,157 @@ public virtual Response<SearchSuggestion[]> GetSearchSuggestions(string searchSt
             }
         }
 
+        private AsyncPageable<QueryResultPage> QueryEventsInternalAsync(
```

Comment:
rearranging code for readability

Reply:
The usual convention for C# is to place private methods towards the end of the file, regardless of where they are being called. So private methods usually go towards the bottom of the file.
It seems like StyleCop does it this was as well:
https://documentation.help/StyleCop/SA1202.html

Curious to know how the team does things on other projects.

Reply:
Interesting and in a way it makes sense. I don't have a strong opinion and can change it.